File: bannerlayout/src/main/java/com/bannerlayout/annotation/DotsAndTitleSiteMode.java
package com.bannerlayout.annotation;

import android.support.annotation.IntDef;

import com.bannerlayout.widget.BannerLayout;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

/**
 * by y on 2017/1/19.
 */

@IntDef({BannerLayout.CENTER,
        BannerLayout.LEFT,
        BannerLayout.RIGHT})
@Retention(RetentionPolicy.SOURCE)
public @interface DotsAndTitleSiteMode {
}

File: bannerlayout/src/main/java/com/bannerlayout/annotation/DotsAndTitleSiteMode.kt
package com.bannerlayout.annotation

import androidx.annotation.IntDef
import com.bannerlayout.widget.BannerLayout

/**
 * by y on 2017/1/19.
 */

@IntDef(BannerLayout.CENTER, BannerLayout.LEFT, BannerLayout.RIGHT)
@Retention(AnnotationRetention.SOURCE)
annotation class DotsAndTitleSiteMode

